loader/loader.c

```diff
@@ -3031,11 +3030,6 @@ static VkResult read_data_files_in_search_paths(const struct loader_instance *in
     if (NULL != override_path) {
         // Local folder and null terminator
         search_path_size += strlen(override_path) + 2;
-
-        // Add the size of any additional search paths defined in the additive environment variable
-        if (NULL != additional_env) {
-            search_path_size += determine_data_file_path_size(additional_env, 0) + 2;
-        }
     } else if (NULL == relative_location) {
```

Comment:
This branch will never be taken because relative_location is set to `""` when on windows. I noticed this by happenstance earlier when reviewing the code.

Reply:
Good catch.  Removed that path.  It should be handled by the "else" part that was there and it also handles the override value.